runtime/cmd/parse/main.go

```diff
@@ -1,3 +1,4 @@
+//go:build !wasm
```

Comment:
Was this accidentally added?

Reply:
Ah yea, for some reason `make fix-lint` adds this every time I run it, and I must have forgotten to drop the changes before committing this time.